platform/mds/mds-performance-tests/src/test/java/org/motechproject/mds/performance/osgi/MdsStressIT.java

```diff
@@ -133,7 +133,7 @@ private void stressTestDeleting(MotechDataService service) {
         for (Object object : service.retrieveAll()) {
             service.delete(object);
         }
-        Long endTime = (System.nanoTime() - startTime) / 1000000;
+        double endTime = (System.nanoTime() - startTime) / 1000000;
 
         LOGGER.info("MDS Service: Deleting " + TEST_INSTANCES + " instances took " + endTime + "ms.");
         logToFile(endTime);
```

Comment:
here you should convert from Long to double, generally maybe it will be better to have two logToFile methods, one with double parameter and second with long, however I think it doesn't really matter. So just convert from Long to double